From c49b4c4acb3b4c6da909b3c4175463cb5714283e Mon Sep 17 00:00:00 2001
From: worldofpeace <worldofpeace@users.noreply.github.com>
Date: Tue, 17 Jul 2018 07:10:58 -0400
Subject: [PATCH 1/1] sbin -> bin

A hook does this but after wrapGAppsHook so the files never get wrapped.
---
 src/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/meson.build b/src/meson.build
index 3d27683..c7ea362 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -9,7 +9,7 @@ config_header = configure_file (
     output: 'config.vala',
     configuration: conf_data
 )
-install_path = join_paths(get_option('prefix'), 'sbin')
+install_path = join_paths(get_option('prefix'), 'bin')
 
 executable(
     meson.project_name(),
-- 
2.17.1

